fuzz_lightyear/usage.py

```diff
@@ -104,6 +104,13 @@ def parse_args(argv: Optional[List[str]] = None) -> argparse.Namespace:
         ),
     )
 
+    parser.add_argument(
+        '--depth',
+        default=6,
```

Comment:
Is there a reason for defaulting this to 6?

Reply:
No specific reason. Just a value from what I've seen in generic request payloads. Was not too sure of this; so provided a way to pass it as argument.